zanata-war/src/main/java/org/zanata/rest/service/ResourceUtils.java

```diff
@@ -162,6 +162,8 @@ boolean transferFromTextFlows(List<TextFlow> from, HDocument to,
                         Response.status(Status.BAD_REQUEST)
                                 .entity("encountered TextFlow with duplicate ID "
                                         + tf.getId()).build();
+                log.error("encountered TextFlow with duplicate ID "
+                        + tf.getId());
```

Comment:
If this is just telling us about an error in user input, it should be WARN at most.